docs/data/material/getting-started/templates/checkout/PaymentForm.js

```diff
@@ -124,22 +153,6 @@ export default function PaymentForm() {
           </Card>
           <Card
             sx={(theme) => ({
-              maxWidth: { sm: '100%', md: '50%' },
-              flexGrow: 1,
-              border: '1px solid',
-              borderColor: 'divider',
-              '&:hover': {
-                background:
-                  theme.palette.mode === 'light'
-                    ? 'linear-gradient(to bottom right, hsla(210, 100%, 97%, 0.5) 25%, hsla(210, 100%, 90%, 0.3) 100%)'
-                    : 'linear-gradient(to right bottom, hsla(210, 100%, 12%, 0.2) 25%, hsla(210, 100%, 16%, 0.2) 100%)',
-                borderColor:
-                  theme.palette.mode === 'light' ? 'primary.light' : 'primary.dark',
-                boxShadow:
-                  theme.palette.mode === 'light'
-                    ? '0px 2px 8px hsla(0, 0%, 0%, 0.1)'
-                    : '0px 1px 8px hsla(210, 100%, 25%, 0.5) ',
-              },
               ...(paymentType === 'bankTransfer' && {
```

Comment:
Why not run these through the styled function?

Reply:
This is the outdated code, all these styles are now in the `PaymentContainer` as a styled component.

Reply:
No, I mean, the styles  applied when `paymentType === 'bankTransfer'`

Reply:
I thought it would end up being too complicated just for the sake of being in the styled component. Currently, it's easy to see the styles applied when it's selected—and closer to context. Is it bad?